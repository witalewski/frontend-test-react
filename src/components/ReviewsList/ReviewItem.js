import React from 'react';

export const ReviewItem = ({review}) => {
    const { text } = review;
    return (<div className="review-item">{text}</div>);
}