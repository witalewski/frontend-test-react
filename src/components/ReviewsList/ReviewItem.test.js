import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import MOCK_REVIEWS from '../../fixtures/reviews.json';
import { ReviewItem } from './ReviewItem';

describe('ReviewItem', () => {
  it('matches snapshot', () => {
    const renderer = new ShallowRenderer();
    renderer.render(
      <ReviewItem review={MOCK_REVIEWS.reviews[0]} />
    );
    const result = renderer.getRenderOutput();

    expect(result).toMatchSnapshot();
  });
});
