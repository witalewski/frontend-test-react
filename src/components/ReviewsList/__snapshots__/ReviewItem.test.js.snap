// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReviewItem matches snapshot 1`] = `
<div
  className="review-item"
>
  <div
    className="review-item__author"
  >
    <img
      alt="Elisa S."
      className="review-item__author-image"
      src="https://s3-media2.fl.yelpcdn.com/photo/u5WL1l-m8edhFJKvxnHvjQ/o.jpg"
    />
    <div>
      <h3>
        Elisa S.
      </h3>
      <p
        className="review-item__date"
      >
        6/5/2017
      </p>
    </div>
  </div>
  <div
    className="review-item__content"
  >
    <StarRating
      rating={5}
    />
    <p>
      It was super delicious  with a wide range of vegan options. Thank you so much bagels & friends for having us
    </p>
  </div>
</div>
`;
